rpclib/project/Common.scala

```diff
@@ -2,7 +2,7 @@ import sbt.Keys._
 
 object Common {
 
-  private final val RpclibVersion = "1.0.0"
+  private final val RpclibVersion = "2.0.0-SNAPSHOT"
```

Comment:
nit: I've generally used to the community convention of making the first version out into the wild "0.1"

Reply:
This has been out for a while, and we also have our own private copy that is 1.x.x